intelligent-service/src/quadraticcontroller.py

```diff
@@ -19,9 +23,52 @@ def quadratic():
             coeff_b = float(arg_b)
             coeff_c = float(arg_c)
         except ValueError:
-            return {"message":"At least a parameter is not a number"},406
+            return {"message": "At least a parameter is not a number"}, 406
         if coeff_a == 0:
-            return {"message":"It is linear"}, 406
+            return {"message": "It is linear"}, 406
     else:
-        return {"message":"Missing parameter"}, 406
+        return {"message": "Missing parameter"}, 406
     return quadratic_service.solve_quadratic(coeff_a, coeff_b, coeff_c), 200
+
+
+@quadratic_api.route('/qe/async/solve', methods=['POST'])
+def async_quadratic():
+    """Returns the acceptance of the async quadratic equation solver task"""
+    if not request.is_json:
+        return {"message": "Wrong format"}, 406
+    content = request.get_json()
+    print(content)
+    if len(content.get("payload")) == 3:
```

Comment:
Negate the condition to increase the readability. Else branch is a single return far from here.
```
if length != 3
 return error
```

This way there is no need for else branch and it is easier to read